ext/approval/info.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * Name: Trash
+ * Author: Matthew Barbour <matthew@darkohlme.net>
+ * Description: Provides "Trash" or "Recycle Bin"-type functionality, storing delete images for later recovery
+ * Documentation:
```

Comment:
These seem to need updating~ (Do we even need metadata in comments, now that we have metadata in `ExtensionInfo`?)

Reply:
I would think not. We can probably just go ahead and delete them all.